config/app.json

```diff
@@ -1,6 +1,11 @@
 {
     "default":{
-        "port":50001
+        "port":50001,
+        "mysql":{
+            "host":"localhost",
+            "user":"username",
+            "password":"password"
+        }
```

Comment:
I recommend moving this to the hinyun object since this username and password is specific to your local machine. Each user will need to add this config to their local object with their own username and password.